src/cast_core/src/CollectionScanner.cpp

```diff
@@ -91,12 +92,15 @@ struct CollectionScanner::PhaseConfig {
           scanSizeBytes{context["ScanSizeBytes"].maybe<IntegerSpec>().value_or(0)},
           collectionSkip{context["CollectionSkip"].maybe<IntegerSpec>().value_or(0)},
           scanDuration{context["ScanDuration"].maybe<TimeSpec>()},
+          scanContinuous{context["ScanContinuous"].maybe<bool>().value_or(false)},
           selectClusterTimeOnly{context["SelectClusterTimeOnly"].maybe<bool>().value_or(false)},
           findOptions{context["FindOptions"].maybe<mongocxx::options::find>().value_or(mongocxx::options::find{})} {
         // The list of databases is comma separated.
         std::vector<std::string> dbnames;
-        boost::split(dbnames, databaseNames, [](char c) { return (c == ','); });
-        for (const auto& dbname : dbnames) {
+        boost::split(dbnames, databaseNames,boost::is_any_of(","));
+
+        for (auto& dbname : dbnames) {
+            boost::trim(dbname);
```

Comment:
@vrachev This line fixes the CollectionScanner.yml leading whitespace in database names issue.